src/plugins/telemetry/telemetry_impl.cpp

```diff
@@ -424,6 +439,24 @@ void TelemetryImpl::process_mount_orientation(const mavlink_message_t &message)
     }
 }
 
+void TelemetryImpl::process_imu_reading_ned(const mavlink_message_t &message)
+{
+    mavlink_highres_imu_t highres_imu;
+    mavlink_msg_highres_imu_decode(&message, &highres_imu);
+    set_imu_reading_ned(Telemetry::IMUReadingNED({highres_imu.xacc,
+                                                  highres_imu.yacc,
+                                                  highres_imu.zacc,
+                                                  highres_imu.xgyro,
+                                                  highres_imu.ygyro,
+                                                  highres_imu.zgyro}));
```

Comment:
I wonder if we should also add mag readings while we're at it.

Reply:
That could be a good idea. I have just found that temperature is also requested as per this issue(#651). That could be a thing too.

Reply:
Yes that would also be helpful.

Reply:
I will get right on it!